_posts/2017-05-02-newsletter.md

```diff
@@ -0,0 +1,238 @@
+---
+layout: newsletter
+title: "Newsletter #8 - Open up the Windows"
+category: newsletter
+permalink: /news/2017/05/
```

Comment:
Change this to `/news/2017/12/` and rename the file to `_posts/2017-12-15-newsletter.md`.  After updating the stats and squashing as you see fit, I'll merge it.

Reply:
Ok, I'll find some time this weekend.